miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/SampleBulkCreateIT.java

```diff
@@ -57,7 +1172,190 @@ public void testCreateIdentityDropdowns() throws Exception {
     table.enterText(Columns.SAMPLE_TYPE, 0, "GENOM");
     assertEquals("GENOMIC", table.getText(Columns.SAMPLE_TYPE, 0));
 
-    // TODO: check other columns
+    List<String> projects = table.getDropdownOptions(Columns.PROJECT, 0);
+    assertTrue(projects.size() > 0);
+    assertTrue(projects.contains("PRO1"));
+
+    table.enterText(Columns.PROJECT, 0, "PRO1");
+    assertEquals("PRO1", table.getText(Columns.PROJECT, 0));
+
+    List<String> donorSexes = table.getDropdownOptions(Columns.DONOR_SEX, 0);
+    assertEquals(5, donorSexes.size());
+    assertTrue(donorSexes.contains("Female"));
+    assertTrue(donorSexes.contains("Unspecified"));
+
+    table.enterText(Columns.DONOR_SEX, 0, "Unspe");
+    assertEquals("Unspecified", table.getText(Columns.DONOR_SEX, 0));
+
+    List<String> qcStatuses = table.getDropdownOptions(Columns.QC_STATUS, 0);
+    assertEquals(10, qcStatuses.size());
+    assertTrue(qcStatuses.contains("Ready"));
+    assertTrue(qcStatuses.contains("Refused Consent"));
+
+    table.enterText(Columns.QC_STATUS, 0, "Rea");
+    assertEquals("Ready", table.getText(Columns.QC_STATUS, 0));
+  }
+
+  @Test
+  public void testCreateIdentityDependencyCells() throws Exception {
+    // Goal: ensure that cells which depend on other columns are updated once the other columns are updated
+    BulkSamplePage page = getCreatePage(1, null, identityClassId);
+    HandsOnTable table = page.getTable();
+
+    table.enterText(Columns.QC_NOTE, 0, "invisible");
+    assertTrue("note is read-only", isStringEmptyOrNull(table.getText(Columns.QC_NOTE, 0)));
+
+    table.enterText(Columns.QC_STATUS, 0, "Okd by Collaborator");
+    table.enterText(Columns.QC_NOTE, 0, "writable note");
+    assertEquals("note is writable", "writable note", table.getText(Columns.QC_NOTE, 0));
+  }
+
+  @Test
+  /**
+   * Need to run these tests after the other sample creates, because sample alias generation
+   * (specifically, SampleNumberPerProject tracking) gets screwed up with manual identity creation.
```

Comment:
Should probably avoid this problem in a different way as ordering of tests isn't guaranteed, and other test classes may end up interfering too. Could use a different project for these samples, or manipulate `SampleNumberPerProject` via Hibernate at the start of test